_posts/2023-11-09-routines.md

```diff
@@ -0,0 +1,85 @@
+---
+layout: post
+title: "Share your best Trino SQL routine"
+author: "Martin Traverso, Dain Sunstrom, David Phillips, Manfred Moser"
```

Comment:
Typo: Sunstrom -> Sundstrom

Reply:
Wow .. I thought I cut and paste that from elsewhere.. now had to check .. did not find any other offenders..